DOL.WHD.Section14c.Web/src/modules/components/sectionEmployer/sectionEmployerTemplate.html

```diff
@@ -150,7 +150,7 @@ <h2>Employer</h2>
             <input class="usa-input-medium" id="mailingAddressZipCode" name="mailingAddressZipCode" type="text" mask="99999-?9?9?9?9?" pattern="[\d]{5}(-[\d]{4})?" data-grouplength="5,4" data-delimiter="-" data-politespace ng-model="formData.employer.mailingAddress.zipCode">
           </div>
 
-          <div ng-class="validate('employer.mailingAddress.county') ? 'usa-input-error' : ''">
+          <div ng-hide="formData.employer.mailingAddress.state === 'DC' || formData.employer.mailingAddress.state === 'AS' || formData.employer.mailingAddress.state === 'GU' || formData.employer.mailingAddress.state === 'MP' || formData.employer.mailingAddress.state === 'PR' || formData.employer.mailingAddress.state === 'UM' || formData.employer.mailingAddress.state === 'VI'" ng-class="validate('employer.mailingAddress.county') ? 'usa-input-error' : ''">
```

Comment:
Instead of having DC and the territories all listed here in the template twice, how about a method in the controller that does this check?  Something like `ng-hide="hasNoCounty(formData.employer.mailingAddress.state)"`, where `hasNoCounty()` does the `indexOf` check against the `$scope.territoriesAndDistricts` array?

Reply:
I tried this originally but was having some trouble with a similiar implementation of hasNoCounty(formData.employer.physicalAddress.state) - when logging the argument in the controller, state was always coming back as undefined.

Reply:
Huh.  Weird!  Okay then.  😆